cnf-certification-test/preflight/suite.go

```diff
@@ -39,7 +39,7 @@ var _ = ginkgo.Describe(common.PreflightTestKey, func() {
 	ginkgo.ReportAfterEach(results.RecordResult)
 
 	// Add safeguard against running the tests if the docker config doesn't exist.
-	if env.GetDockerConfigFile() == "" {
+	if env.GetDockerConfigFile() == "" || env.GetDockerConfigFile() == "NA" {
```

Comment:
Shouldn't we check if the file exists instead?

Reply:
Yeah that's done in the script prior to kicking off the suite.  The bug in this case is just not handling the "NA" that we purposefully add when the dockerconfig is missing.

https://github.com/test-network-function/cnf-certification-test/blob/main/run-tnf-container.sh#L141-L158